tests/python/gaia-ui-tests/gaiatest/apps/system/regions/activities.py

```diff
@@ -53,6 +53,8 @@ def tap_camera(self):
         camera = Camera(self.marionette)
         self.wait_for_condition(lambda m: self.apps.displayed_app.name == camera.name)
         self.apps.switch_to_displayed_app()
+        camera.wait_for_loading_spinner_displayed()
+        camera.wait_for_loading_spinner_hidden()
```

Comment:
I suggest moving these 2 lines in ```wait_for_capture_ready()```. The way they will run in all of the instances where we use the camera app. We should test this thoroughly so that we don't break other tests.